### ISSUE TITLE
Plugin Activation Message Not Announced

### ISSUE THREAD

**Issue by avaaec** [pressbooks/pressbooks#1787]
### Prerequisites

* [X] Can you reproduce the behavior you wish to address with [all other plugins deactivated](http://codex.wordpress.org/Multisite_Network_Administration#Plugins)?
* [X] Are you running the [latest version of Pressbooks](http://github.com/pressbooks/pressbooks/releases/latest/) and the [latest version of WordPress](http://codex.wordpress.org/Upgrading_WordPress)?
* [X] Are you experiencing this issue with one of our built-in root or book themes?
* [X] Did you [perform a cursory search](https://github.com/issues?q=+is%3Aissue+repo%3Apressbooks%2Fpressbooks) to see if your bug or enhancement is already listed?

### Steps to Reproduce

1. Go to the admin page for a book
2. Navigate to the Plugins page
3. In the table of plugins find a plugin (example H5P)
4. Select the activate link in the "Plugin" column of that plugin's row
5. Notice that the page reloads and that a message from the plugin is added to the page above the table
6. Notice that a message saying "Plugin Activated" is added below the message added by the plugin

**Expected behavior:** 
When visual notifications are given, non-visual notifications should be given as well. In this case the "Plugin Activated" message must be announced to non-visual users using aria-live or equivalent. 

The message from the plugin is longer and can contain links so it won't work well as a aria-live announcement. A solution to this is to give an aria-live or equivalent announcement that says "New message from plugin under the Plugin heading" or something to that effect. This way non-visual users are aware that this information has appeared and won't miss anything.
**Actual behavior:** 
Currently these messages are not announced. Additionally, because the page refreshes when you add a plugin, non-visual users are unlikely to explore the area of the page they appear on, making them virtually invisible.

#### Book Info

Book ID: 207
Book URL: https://uw.pressbooks.pub/accessibilitytesting/
Book Privacy: Public

#### Browser

Platform: Windows
Browser Name: Firefox
Browser Version: 60.0
User Agent String: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:60.0) Gecko/20100101 Firefox/60.0

#### WordPress Configuration

Network URL: http://uw.pressbooks.pub/
Network Type: Subdirectory
Version: 5.2.2
Language: en_US
WP_ENV: production
WP_DEBUG: Disabled
Memory Limit: 96M

#### Pressbooks Configuration

Version: 5.9.2
Book Theme: McLuhan
Book Theme Version: 2.8.10
Root Theme: Aldine
Root Theme Version: 1.6.4

#### Pressbooks Dependencies

Epubcheck: Installed
Kindlegen: Installed
xmllint: Installed
PrinceXML: Not Installed
Saxon-HE: Installed

#### Must-Use Plugins

Bedrock Autoloader: 1.0.0
hm-autoloader.php: n/a
telemetry.php: n/a

#### Network Active Plugins

Hypothesis: 0.6.0
Pressbooks: 5.9.2
Pressbooks BISAC: 2.0.0
Pressbooks LTI Provider: 1.3.1
Pressbooks Network Analytics: 1.1.0
Pressbooks Stats: 1.6.3
Pressbooks Whitelabel: 1.4.0
Redis Object Cache: 1.4.3
User Activation Keys: 4.6
WP Fail2Ban Redux: 0.5.0

#### Book Active Plugins

Hypothesis: 0.6.0
Pressbooks: 5.9.2
Pressbooks BISAC: 2.0.0
Pressbooks LTI Provider: 1.3.1
Pressbooks Network Analytics: 1.1.0
Pressbooks Stats: 1.6.3
Pressbooks Whitelabel: 1.4.0
Redis Object Cache: 1.4.3
User Activation Keys: 4.6
WP Fail2Ban Redux: 0.5.0

#### Inactive Plugins

Akismet Anti-Spam: 4.1.2
H5P: 1.14.1
Parsedown Party: 1.2.0
TablePress: 1.9.2
WP QuickLaTeX: 3.8.6

#### Server Configuration

PHP Version: 7.1.31-1+ubuntu16.04.1+deb.sury.org+1
MySQL Version: 5.5.5
Webserver Info: nginx/1.15.8

#### PHP Configuration

Safe Mode: Disabled
Memory Limit: 256M
Upload Max Size: 25M
Post Max Size: 25M
Upload Max Filesize: 25M
Time Limit: 240
Max Input Vars: 1000
URL-aware fopen: On (1)
Display Errors: N/A

#### PHP Extensions

OPcache: Zend
XDebug: Disabled
cURL: Supported
cURL Version: 7.47.0
imagick: Not Installed
xsl: Installed

**Comment by SteelWagstaff** [comment]
Hi @avaaec, thanks for filing this issue. It appears to me that most plugins when activated/deactivated display a dismissable notice at the top of the page (i.e.):
![Screenshot from 2019-10-18 16-41-54](https://user-images.githubusercontent.com/13485451/67130243-4da52d00-f1c6-11e9-9bf2-5fa487d89900.png)
![Screenshot from 2019-10-18 16-42-16](https://user-images.githubusercontent.com/13485451/67130226-45e58880-f1c6-11e9-84c7-a6f06a33e305.png)

This behavior is inherited from core WordPress and looks like well-formed accessible HTML to me: 
```<div id="message" class="updated notice is-dismissible" role="status"><p>Plugin <strong>deactivated</strong>.</p><button type="button" class="notice-dismiss"><span class="screen-reader-text">Dismiss this notice.</span></button></div>```

Second, the additional message that appears upon activation of H5P is generated by the H5P plugin itself:
![Screenshot from 2019-10-18 16-42-04](https://user-images.githubusercontent.com/13485451/67130436-e340bc80-f1c6-11e9-901f-535bafee6925.png)

Like the more generic message, this also includes a role="status" attribute: 
```<div class="updated" role="status"><p>Thank you for choosing H5P.</p><p>You are ready to <a href="https://integrations.pressbooks.network/portuguese/wp-admin/admin.php?page=h5p_new">start creating</a> interactive content. Check out our <a href="https://h5p.org/content-types-and-applications" target="_blank">Examples and Downloads</a> page for inspiration.</p><p>If you need any help you can always file a <a href="https://wordpress.org/support/plugin/h5p" target="_blank">Support Request</a>, check out our <a href="https://h5p.org/forum" target="_blank">Forum</a> or join the conversation in the <a href="https://gitter.im/h5p/CommunityChat" target="_blank">H5P Community Chat</a>.</p></div>```

My understanding is that 
> The status role is a type of live region and a container whose content is advisory information for the user that is not important enough to justify an alert, and is often presented as a status bar. When the role is added to an element, the browser will send out an accessible status event to assistive technology products which can then notify the user about it.
> Status information content must be provided within a status object, and it should be ensured that this object does not receive focus. If another part of the page controls what appears in the status, the relationship should be made explicit via the aria-controls attribute.

Futhermore, "Assistive technology products should listen for such an event and notify the user accordingly" [source](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_status_role)

As such, I'm not seeing that there's a problem here and am going to close this issue. 

Furthermore, Pressbooks does not generate either of these messages (the first is inherited from core WordPress, and the second is generated by the H5P plugin itself). If you feel that these messages ought to be given a `role="alert"` value rather than `role="status"`, that's something that would be better filed upstream. 

If you feel I'm mistaken, please feel free to share more context in a comment here and we'll consider reopening.

**Comment by avaaec** [comment]
Under review by UW ATS
